src/app/home/countdown/countdown.component.html

```diff
@@ -0,0 +1,81 @@
+<div *ngIf="!isFinishDate" class="hide-on-small-only">
+  <div class="row container">
+    <div class="col s12 center-align">
+      <h2 id="countdown-title" class="medium-text"> {{'countdown.title' | translate}} </h2>
+    </div>
+  </div>
+  <div class="row container">
+    <div class="col s3">
+      <div class="card">
+        <div class="card-image waves-effect waves-block waves-light">
+          <p class="medium-text center-align"> {{daysRemaining | appNumberTransform}} </p>
```

Comment:
```suggestion
          <p class="medium-text center-align">{{ daysRemaining | appNumberTransform }}</p>
```

Reply:
In all the places where you bind data, try using the format that I'm suggesting in the above comment, this is to keep the code under a standard.